CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ Please note we have a code of conduct, please follow it in all your interactions
 2. Create a branch to begin work on either using the [GitHub UI](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository), or via your [command line](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging).
 3. Checkout the branch locally and work on your issue.
 4. When you are finished on your feature/issue, push it to the remote repository and [create your pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
-5. [Request a review](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/requesting-a-pull-request-review) from one of the reviewers listed in the reviewers section of this document.
+5. [Request a review](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/requesting-a-pull-request-review) from one of the reviewer's listed in the reviewer's section of this document.
```

Comment:
This is grammatically incorrect. `'s` indicates ownership, whereas an individual reviewer does not own the concept of "listed" nor the section of the document being referenced. 

"Reviewers" here is a concept in itself, and should not use `'s`